File: 01_mobile_app/front/kotlin/app00/app/src/main/java/com/example/app00/model/FoodShop.kt
package com.example.app00.model

import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.outlined.Email
import androidx.compose.ui.graphics.vector.ImageVector

class FoodShop {

    val id: String
    val name: String
    val genre: String?
    val rating: Double?
    val lastUsedDate: String?
    val memo: String?

    constructor(id: String, name: String, genre: String?, rating: Double?, lastUsedDate: String?, memo: String?) {
        this.id = id
        this.name = name
        this.genre = genre
        this.rating = rating
        this.lastUsedDate = lastUsedDate
        this.memo = memo
    }

}

File: 01_mobile_app/front/reactnative/app00/model/FoodShop.ts
export class FoodShop {

  public id: string;
  private name: string;
  private genre: string;
  private lastUsedDate: string;
  private rating: string;
  private memo: string;

  constructor(id: string, name: string, genre: string, rating: string, lastUsedDate: string, memo: string) {
    this.id = id;
    this.name = name;
    this.genre = genre;
    this.lastUsedDate = lastUsedDate;
    this.rating = rating;
    this.memo = memo;
  }

}
